### PR TITLE
Add Threshold Network Token (T) to Superchain Token List

### PR DESCRIPTION
**Pull request by sap57624**
**Description**

Add Threshold Network Token (T) to Superchain Token List.

### CODE REVIEW

**Comment by zainbacchus** [comment]
Hey thanks for the PR! We'll review and follow up accordingly :)

File: data/T/data.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "tokens": {
+    "ethereum": {
+      "address": "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5"
+    },
+    "optimism": {
+      "address": "0x747e42Eb0591547a0ab429B3627816208c734EA7"
+    },
+    "goerli": {
+      "address": "0x3f16380656cAE45D3f80D8833682d2b606eD094A"
+    }
```
Comment (jakim929) [inline data/T/data.json:17]: Is this token being bridge to optimism-goerli as well? if possible, can you add an address for optimism-goerli?

File: data/T/data.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "decimals": 18,
+  "description": "Threshold Network is the home of $tBTC, proxy re-encryption #PRE, $thUSD, and all threshold cryptography.",
+  "website": "https://threshold.network",
+  "twitter": "@TheTNetwork",
+  "tokens": {
+    "ethereum": {
+      "address": "0xCdF7028ceAB81fA0C6971208e83fa7872994beE5"
+    },
+    "optimism": {
+      "address": "0x747e42Eb0591547a0ab429B3627816208c734EA7"
```
Comment (jakim929) [inline data/T/data.json:13]: This contract doesn't seem to be verified on optimism etherscan yet. Can you verify this contract?

**Review by jakim929 — CHANGES_REQUESTED** [review]
Thank you for your PR. I apologize for the delay!

Requesting changes regarding op-goerli deployment and contract verification on optimism.

File: data/T/data.json (hunk 2 above)
Comment (sap57624) [reply data/T/data.json:13]: Hi @jakim929, thanks for looking here.

I can't seem to verify the contract, and receive this: "Error! Unable to generate Contract ByteCode and ABI" 

Do you think you can take a look please? 

It seems that the Optimism Bridge 3 is behaving differently after the fork, and creating additional internal transactions to what it did before. The byte code for our contract deployment is different to those prior to the fork. Could this be the reason the verification fails?

Here is the L2 ERC20 token contract creation: https://optimistic.etherscan.io/tx/0xad69e07c124ada374d23131333c9d892d64559488fbc85df44751adf76b66981

File: data/T/data.json (hunk 1 above)
Comment (sap57624) [reply data/T/data.json:17]: There are not currently plans to bridge to Goerli.

**Comment by sap57624** [comment]
HI @jakim929, the contract has successfully been verified, and we do not intend to bridge to Goerli at this stage.

https://optimistic.etherscan.io/address/0x747e42eb0591547a0ab429b3627816208c734ea7

Please let me know if there is anything else we need to complete to get this merged.

Thank you.

File: data/T/data.json (hunk 2 above)
Comment (sap57624) [reply data/T/data.json:13]: Contract is now verified.

https://optimistic.etherscan.io/address/0x747e42eb0591547a0ab429b3627816208c734ea7

**Review by sap57624 — COMMENTED** [review]
HI @jakim929, the contract has successfully been verified, and we do not intend to bridge to Goerli at this stage.

https://optimistic.etherscan.io/address/0x747e42eb0591547a0ab429b3627816208c734ea7

Please let me know if there is anything else we need to complete to get this merged.

Thank you!

File: data/T/data.json (hunk 2 above)
Comment (jakim929) [reply data/T/data.json:13]: thank you!

**Review by jakim929 — APPROVED** [review]

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**